content/millennium/r4/medications/medication_request.md

```diff
@@ -0,0 +1,144 @@
+---
+title: MedicationRequest | R4 API
+---
+
+# MedicationRequest
+
+* TOC
+{:toc}
+
+## Overview
+
+The Medication Request resource provides orders for all medications along with administration instructions for a
```

Comment:
it seems for non-med resources we do one word (e.g. AllergyIntolerance) but for meds we have separated them. I'd prefer to use the FHIR name so MedicationRequest throughout

Reply:
https://github.com/cerner/fhir.cerner.com/pull/332/commits/909f357d1c858d588741c6b3265da1827c6a43ce?file-filters%5B%5D=.md#diff-91cabf44fbcf0c5aaa861d340e57d246R12